### PR TITLE
JBDS-3716 Improve all in one installer performance

### PR DESCRIPTION
**Pull request by dgolovin**
Result is new bundled installer starts 3 times faster than old one, but it is 70Mb bigger.

### CODE REVIEW

File: gulp-tasks/config.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ let config = {
 config.buildFolderRoot = path.join('dist', config.artifactPlatform + '-' + config.artifactArch );
 config.buildFileNamePrefix = config.artifactName;
 config.buildFolderPath = path.resolve(config.buildFolderRoot);
+config.genFolderPath = path.join(config.buildFolderPath, 'devsuite-win32-x64');
```
Comment (jrichter1) [inline gulp-tasks/config.js]: This variable looks like it is not being used at all.

**Comment by jrichter1** [comment]
The blazing speed at which it unpacks is awesome. And it is still roughly the same size as before (without compression??).

File: gulp-tasks/config.js (hunk 1 above)
Comment (dgolovin) [inline gulp-tasks/config.js]: Right, I was trying to skip app generation every time I run the build and forget to clean up. 
Another reason for it not to be here is it is windows specific.
I'll remove it.
Thanks @jrichter1!

**Comment by dgolovin** [comment]
I can make it even the same size, but that would require to pack every bundled binary and then unpack it first before install. I think we can pay 70Mb in size gain for not doing that :)

**Comment by dgolovin** [comment]
test this

**Comment by jrichter1** [comment]
test this

**Comment by jrichter1** [comment]
test this

**Pull request merged**